tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1530,3 +1534,75 @@ function assertIncludesMessage(messageArr, message) {
     return false
   }
 }
+
+Then('the decline action button should be visible on the shared-with-me page', async function () {
+  const declineBatchActionButtonVisibility =
+    await sharedWithMePage.isDeclineBatchButtonVisibleForAutoAccept()
+  assert.ok(
+    declineBatchActionButtonVisibility,
+    `Expected decline action button to be visible but was not`
+  )
+})
+
+Then(
+  'the delete action button should be not visible on the shared-with-others page',
+  async function () {
+    let deleteBatchActionButtonVisibility =
+      await sharedWithOthersPage.isDeleteBatchButtonVisibleForAutoAccept()
+    if (deleteBatchActionButtonVisibility === -1) {
```

Comment:
it is a little vague to use `-1` here, can you parse the return value as a boolean in the `isDeleteBatchButtonVisibleForAutoAccept` function?

Reply:
Yes, we can use if function inside isDeleteBatchButtonVisibleForAutoAccept() in place of filesContext.js .There is no built-in function for parsing -1 into boolean value.

Reply:
Please do return a boolean from `isDeleteBatchButtonVisibleForAutoAccept`. Reading the function name suggests that it returns a boolean. It's unexpected that it doesn't.